app/components/jobseekers/search_results/heading_component.rb

```diff
@@ -5,28 +5,28 @@ def initialize(vacancies_search:)
     @location = @vacancies_search.location_search.location
     @polygon_boundaries = @vacancies_search.location_search.polygon_boundaries
     @radius = @vacancies_search.search_criteria[:radius]
-    @total_count = @vacancies_search.total_count
+    @total_count = number_with_delimiter(@vacancies_search.total_count)
   end
 
   def heading
     if @keyword.present? && @polygon_boundaries.present?
-      t("jobs.search_result_heading.keyword_location_polygon_html",
-        jobs_count: number_with_delimiter(@total_count), location: @location, keyword: @keyword, count: @total_count, radius: @radius, units: t("jobs.search_result_heading.unit_of_length").pluralize(@radius.to_i))
+      t("jobs.search_result_heading.keyword_location_polygon_html", jobs_count: @total_count, location: @location, keyword: @keyword, count: @total_count, radius: @radius, units: units)
```

Comment:
Just realised I need to fix this, jobs_count and count need to take different inputs